frontend/src/Admin/components/ShowSubmissionDetails.js

```diff
@@ -75,16 +80,25 @@ const renderSubmissionByType = submission => {
         return null
       } else if (submission.path.endsWith('.jpg')) {
         return (
+          <Fragment>
           <PhotoThumbnail
             alt={submission.title}
             src={`${STATIC_PATH}${getImageThumbnail(submission.path)}`}
           />
+          <div className='text-left'>
+            <a href={`${STATIC_PATH}${submission.path}`} target='_blank'>
+              <Button color='primary'>View Image</Button>
+            </a>
+          </div>
+          </Fragment>
         )
       } else if (submission.path.endsWith('.pdf')) {
         return (
-          <a href={`${STATIC_PATH}${submission.path}`} target='_blank'>
-            <FaPDF /> View PDF
-          </a>
+          <div className='text-left'>
+            <a href={`${STATIC_PATH}${submission.path}`} target='_blank'>
+              <Button color='primary'>View PDF</Button>
+            </a>
+          </div>
```

Comment:
got rid of the icon that was breaking it and turned it into a button so it looks nicer than just a plain link (also I think this is more consistent anyway)